### PR TITLE
smearing_method parsed

### PR DESCRIPTION
**Pull request by Mdmorshadurrahman**
image
after rebase, I started working but on this portion of coding showed an error like this.
image
then for testing my written code I commented on those codes and after testing successfully uncommented those and requested pull. can you check is it an error or not.

### CODE REVIEW

**Comment by ondracka** [comment]
This is OK, but the tests are missing, I suppose you have them somewhere, but they are not in this pull request. Add the tests and than this is good to go.

BTW good job on the rebasing, this now looks fine.

**Comment by ondracka** [comment]
Regarding the missing _file_handler, you are probably using and old nomad-lab python package. You need nomad-lab v0.10.3, than this should be fixed.

File: tests/test_parser.py [hunk 1]
```diff
@@ -38,6 +38,10 @@ def Ha_to_J(value):
     return (value * units.hartree).to_base_units().magnitude
 
 
+def K_to_J(value):
+    return (value * units.joules).to_base_units().magnitude
```
Comment (ondracka) [inline tests/test_parser.py:42]: I think it should be units.kelvin here. What I don't understand though is why the test pass?

**Review by ondracka — COMMENTED** [review]
I need to think about what is going on with the unit conversion, however I can commit this as a follow up, so just take the next task and I'll finish this one

File: tests/test_parser.py (hunk 1 above)
Comment (Mdmorshadurrahman) [reply tests/test_parser.py]: is it supposed to convert kelvin to joules or vice-versa?

**Pull request merged**